topics/admin/tutorials/ansible-galaxy/tutorial.md

```diff
@@ -432,6 +668,12 @@ The configuration is quite simple thanks to the many sensible defaults that are
 >    >
 >    {: .question}
 >
+>    > ### {% icon tip %} Miniconda fails to work
+>    > The Galaxy user is created to separate privileges. Then we add `uchida.miniconda`, which is run as the Galaxy user.
+>    >
+>    > The miniconda role, attempts to install tar and bzip2, even when the user doesn't have permissions to do this. This issue has been addressed in the codebase, but, the role has not seen a release which would address this. You do *not* need to run this role, Galaxy will attempt to install conda when it is missing. We added it to prevent a possible race condition between the two mules which both attempt to install conda. So, remove the role from your playbook, and carry on, if you have issues.
```

Comment:
```suggestion
>    > The miniconda role attempts to install tar and bzip2, even when the user doesn't have permissions to do this. This issue has been addressed in the codebase, but, the role has not seen a release which would address this. You do *not* need to run this role, Galaxy will attempt to install conda when it is missing. We added it to prevent a possible race condition between the two mules if both attempt to install conda at the same time. So, remove the role from your playbook, and carry on, if you have issues.
```